docs/blog/2019-07-10-user-testing-accessible-client-routing/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: What we learned from user testing of accessible client-side routing techniques with Fable Tech Labs
+author: Marcy Sutton
+date: 2019-07-10
+excerpt: "We conducted 5 user testing sessions for accessibility research with Fable Tech Labs to determine which client-side routing techniques are the most ergonomic and intuitive to users with disabilities, and if any of the techniques presented barriers detracting from their browsing experience."
+tags:
+  - accessibility
+  - user testing
+  - client-side routing
+  - React
+  - UI
+---
+
+In June 2019, I (Gatsby Head of Learning [Marcy Sutton](https://twitter.com/marcysutton)) conducted 5 user testing sessions for accessibility research with [Fable Tech Labs](https://www.makeitfable.com/), a Toronto-based start-up that’s “making it easier for digital teams to engage people with disabilities in product development.”
```

Comment:
Later on in the post You refer to yourself in 3rd person ("Marcy connected", "Marcy learned") -   should this be consistent?

Reply:
Good point! I wrote it in Google Docs and then started to change my mind once my byline and photo were on there. I'll take a closer look...

Reply:
Updated!

Reply:
Looks great! No more comments from me ;)